dist/angular-auth.min.js

```diff
@@ -1,5 +1,5 @@
 angular.module("PlAuth",["restmod"]);
-!function(){"use strict";function t(){function t(){this.getLocalStoragePrefix=function(){return u},this.getUidHeaderName=function(){return i},this.getTokenHeaderName=function(){return r},this.getUidKey=function(){return e},this.getTokenKey=function(){return n}}var e="uid",n="token",i="X-User-Email",r="X-User-Token",u="angularAuth";this.setLocalStoragePrefix=function(t){u=t},this.setTokenHeaderName=function(t){r=t},this.setUidHeaderName=function(t){i=t},this.setUidKey=function(t){e=t},this.setTokenKey=function(t){n=t},this.$get=function(){return new t}}angular.module("PlAuth").provider("AuthConfig",t)}();
-!function(){"use strict";function e(e,t){function r(){return!!e.getKey("uid")&&!!e.getKey("token")}function n(){return!!e.getKey("uid")}function i(){var t={};return t[g]=e.getKey("uid"),t[d]=e.getKey("token"),t}function o(t){if(!angular.isObject(t))throw new Error("an object was expected to store");if(t[s]){if(!t[c])throw new Error("A uid is need");e.setKey("token",t[s])}t[c]&&e.setKey("uid",t[c])}function u(){e.eraseKey("uid"),e.eraseKey("token")}var a={store:o,clear:u,isLogged:r,isRegistered:n,getAuthorizationHeaders:i},d=t.getTokenHeaderName(),g=t.getUidHeaderName(),s=t.getTokenKey(),c=t.getUidKey();return a}angular.module("PlAuth").service("AuthSrv",e),e.$inject=["LocalDataSrv","AuthConfig"]}();
+!function(){"use strict";function t(){function t(){this.getLocalStoragePrefix=function(){return r},this.getUidHeaderName=function(){return i},this.getTokenHeaderName=function(){return u},this.getUidKey=function(){return e},this.getTokenKey=function(){return n},this.getUidRequired=function(){return o}}var e="uid",n="token",i="X-User-Email",u="X-User-Token",r="angularAuth",o=!0;this.setLocalStoragePrefix=function(t){r=t},this.setTokenHeaderName=function(t){u=t},this.setUidHeaderName=function(t){i=t},this.setUidKey=function(t){e=t},this.setTokenKey=function(t){n=t},this.setUidRequired=function(t){o=t},this.$get=function(){return new t}}angular.module("PlAuth").provider("AuthConfig",t)}();
+!function(){"use strict";function e(e,t){function r(){var t=[];return f&&t.push(!!e.getKey("uid")),t.push(!!e.getKey("token")),t.every(function(e){return e===!0})}function n(){return!!e.getKey("uid")}function i(){var t={};return f&&(t[s]=e.getKey("uid")),t[d]=e.getKey("token"),t}function u(t){if(!angular.isObject(t))throw new Error("an object was expected to store");if(t[g]){if(!t[c]&&f)throw new Error("A uid is need");e.setKey("token",t[g])}t[c]&&f&&e.setKey("uid",t[c])}function o(){e.eraseKey("uid"),e.eraseKey("token")}var a={store:u,clear:o,isLogged:r,isRegistered:n,getAuthorizationHeaders:i},d=t.getTokenHeaderName(),s=t.getUidHeaderName(),g=t.getTokenKey(),c=t.getUidKey(),f=t.getUidRequired();return a}angular.module("PlAuth").service("AuthSrv",e),e.$inject=["LocalDataSrv","AuthConfig"]}();
```

Comment:
Line is too long.<br>Confusing use of '!'.<br>Missing semicolon.<br>Expected '{' and instead saw 'throw'.<br>Too many errors. (60% scanned).